src/tools/spin/spinquic.cpp

```diff
@@ -938,7 +940,9 @@ CXPLAT_THREAD_CALLBACK(RunThread, Context)
             ASSERT_ON_NOT(Gb.Buffers[j].Buffer);
         }
 
+        CxPlatLockAcquire(&RunThreadLock);
         QUIC_STATUS Status = MsQuicOpen2(&Gb.MsQuic);
+        CxPlatLockRelease(&RunThreadLock);
```

Comment:
Let's **only** do this is static linking. I want dynamic linking to stress this by calling in parallel. Can you add a #ifdef around it?